examples/with-rollup/package.json

```diff
@@ -1,9 +1,5 @@
 {
   "private": true,
-  "workspaces": [
-    "apps/*",
-    "packages/*"
-  ],
```

Comment:
I think we still want workspaces in this example?

Reply:
It's pnpm! Fixing what shouldn't have been there in the first place.